operators/src/mock/mock_raster_source.rs

```diff
@@ -189,7 +188,9 @@ mod tests {
                 "resultDescriptor": {
                     "dataType": "U8",
                     "spatialReference": "EPSG:4326",
-                    "measurement": "unitless",
+                    "measurement": {
+                        "type": "unitless"
+                    },
```

Comment:
Is this really better in this case?
Maybe "unitless" or { type…} ?

Reply:
it's at least consistent to always have a type